ansible_collections/arista/avd/roles/eos_designs/docs/tables/wan-interfaces-settings.md

```diff
@@ -157,6 +164,12 @@
             # The peer device IPv4 address (no mask). Used as default route gateway if `set_default_route` is true and `ip` is an IP address.
             peer_ip: <str>
 
+            # Configure IPv4 static routes pointing to `peer_ip`.
+            static_routes: # >=1 items
+
+                # IPv4_network/Mask
+              - <str>
```

Comment:
would we wan this to be list of dict instead to add some options?

```
edge1B(config)#ip route 1.1.1.1/32 ?
  A.B.C.D                    IPv4 address of the nexthop router
  A:B:C:D:E:F:G:H            IPv6 address of the nexthop router
  Dps                        DPS Tunnel Interface
  Ethernet                   Ethernet interface
  Fabric                     Fabric Interfaces
  Loopback                   Hardware interface used for looping packets
  Management                 Management interface
  Null0                      Interface that drops all traffic
  Port-Channel               Port-Channel Sub Interface
  Tunnel                     Tunnel interface
  Vlan                       Logical interface into a VLAN
  Vxlan                      VXLAN Tunnel Interface
  egress-vrf                 egress VRF for this route
  fe80::A:B:C:D%<interface>  IPv6 linklocal address of the nexthop router
  vtep                       Configure VXLAN Tunnel End Points
```

```
edge1B(config)#ip route 1.1.1.1/32 2.2.2.2 ?
  community  BGP communities
  metric     Metric for this route
  name       Next hop name
  tag        Route tag
  track      Track this route
  <1-255>    Administrative distance for this route
  <cr>
```

I can see this being intresting at some point

Reply:
done